.travis.yml

```diff
@@ -1,16 +1,31 @@
 language: php
 php:
+  - 7.1
   - 7.2
   - 7.3
   - 7.4
 
+env:
+  global:
+    - RUN_PHPSTAN="FALSE"
+  matrix:
+    - PREFER_LOWEST="" WITH_COVERAGE="--coverage-clover=coverage.xml"
+    - PREFER_LOWEST="--prefer-lowest" $WITH_COVERAGE=""
+
 matrix:
+  include:
+    - name: 'PHPStan'
+      php: 7.2
+      env: RUN_PHPSTAN="TRUE"
   fast_finish: true
 
 before_script:
-  - composer install
+  - composer update --prefer-source $PREFER_LOWEST
```

Comment:
Hmm not sure we need this here. The prefer-lowest build is something which is usefull in the build matrix (additional to the regular „composer install“ build)